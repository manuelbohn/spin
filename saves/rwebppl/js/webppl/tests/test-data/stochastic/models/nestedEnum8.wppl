var logistic = function(z) {
  return 1 / (1 + Math.exp(-z));
};

var model = function() {
  var x = logistic(gaussian(3, 2));
  var marginal = Infer({model() {
    var z = flip(x);
    factor(!z ? 0 : -2);
    return z;
  }});
  var y = sample(marginal);
  return x * y;
};
