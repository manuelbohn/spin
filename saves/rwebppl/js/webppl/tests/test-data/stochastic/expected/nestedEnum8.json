{
  "mean": 0.613,
  "std": 0.457
}
